test/appium/tests/atomic/chats/test_one_to_one.py

```diff
@@ -782,46 +889,40 @@ def test_markdown_support_in_messages(self):
         self.errors.verify_no_errors()
 
 
-class TestMessagesOneToOneChatSingle(SingleDeviceTestCase):
-
-    @marks.testrail_id(5317)
-    @marks.critical
-    def test_copy_and_paste_messages(self):
-        sign_in = SignInView(self.driver)
-        home = sign_in.create_user()
-
-        home.join_public_chat(''.join(random.choice(string.ascii_lowercase) for _ in range(7)))
-        chat = sign_in.get_chat_view()
-        message_text = {'text_message': 'mmmeowesage_text'}
-        formatted_message = {'message_with_link': 'https://status.im',
-                             # TODO: blocked with 11161 (rechecked 23.11.21, valid)
-                             # 'message_with_tag': '#successishere'
-                             }
-        message_input = chat.chat_message_input
-        message_input.send_keys(message_text['text_message'])
-        chat.send_message_button.click()
+@pytest.mark.xdist_group(name="one_to_one_chat_single")
+class TestOneToOneChatSingleSharedDevice(MultipleSharedDeviceTestCase):
 
-        chat.copy_message_text(message_text['text_message'])
+    @classmethod
+    def setup_class(cls):
+        cls.drivers, cls.loop = create_shared_drivers(1)
+        cls.driver = cls.drivers[0]
+        cls.sign_in = SignInView(cls.driver)
+        cls.home = cls.sign_in.create_user()
+        cls.chat = cls.home.add_contact(basic_user['public_key'])
+        cls.chat_name = basic_user['username']
 
-        message_input.paste_text_from_clipboard()
-        if message_input.text != message_text['text_message']:
-            self.errors.append('Message %s text was not copied in a public chat' % message_text['text_message'])
-        message_input.clear()
-
-        for message in formatted_message:
-            message_input.send_keys(formatted_message[message])
-            chat.send_message_button.click()
+    @marks.testrail_id(5387)
+    @marks.high
+    def test_delete_one_to_one_chat_via_delete_button(self):
+        message = self.home.get_random_message()
+        self.chat.send_message(message)
+        self.home.just_fyi("Deleting chat via delete button and check it will not reappear after relaunching app")
+        self.chat.delete_chat()
+        self.chat.get_back_to_home_view()
+
+        if self.home.get_chat_from_home_view(self.chat_name).is_element_displayed():
+            self.errors.append('Deleted %s chat is shown, but the chat has been deleted' % self.chat_name)
+        self.driver.close_app()
+        self.driver.launch_app()
+        self.sign_in.sign_in()
```

Comment:
self.home.reopen_app()

Reply:
Also don't think it is worth to start a separate group only for 1 test, maybe better to add this as the last test into the 2driver group or move it to medium

Reply:
I think we can move other tests to this group too. But it's better not to have tests with 1 driver in the same group with multiple drivers.